ext/ffi/lib.rs

```diff
@@ -351,10 +365,24 @@ struct ForeignFunction {
   result: NativeType,
 }
 
+#[derive(Deserialize, Debug)]
+#[allow(dead_code)]
+struct ForeignStatic {
+  name: Option<String>,
+  r#type: String,
```

Comment:
```suggestion
  #[serde(rename = "type)]
  type_: String,
```
Prefer `type_` over raw literal syntax

Reply:
EDIT: Should be `"type"` instead of `"type`